.changelog/3511.feature.md

```diff
@@ -0,0 +1,3 @@
+go/consensus/api: Add Registry/Staking/Scheduler to ClientBackend
```

Comment:
Nit: Change the title to:
```
go/consensus/api: Add `Registry()`/`Staking()`/`Scheduler()` to client backend
```

(Ideally, I would use `` `ClientBackend` ``, but then the title would be too long.)